README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
```

Comment:
I think a more creative and useful way to convey this information would be as an GH issue template with checkboxes for the various items. Do you want to give that a try?

Reply:
yeah! ill work on adding in all the changes suggested into an issue template. 

From this guide
https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository it appears like we can have different issue templates for the repo.

I like this idea, the onboarding process then becomes visible as a ticket.